admin/partials/nginx-helper-general-options.php

```diff
@@ -620,9 +666,14 @@
 							echo wp_kses(
 								sprintf(
 									'%1$s<br />%2$s<br /><small>%3$s</small>',
-									esc_html__( 'Or,', 'nginx-helper' ), esc_html__( 'Text to manually copy and paste in nginx settings', 'nginx-helper' ), esc_html__( '(if your network is small and new sites are not added frequently)', 'nginx-helper' )
+									esc_html__( 'Or,', 'nginx-helper' ),
+									esc_html__( 'Text to manually copy and paste in nginx settings', 'nginx-helper' ),
+									esc_html__( '(if your network is small and new sites are not added frequently)', 'nginx-helper' )
```

Comment:
This is not correct way to make small string translatable. We need to allow entire string translatable.